src/dialogs/media-browse/show-media-browser-dialog.ts

```diff
@@ -16,12 +16,5 @@ export const showMediaBrowserDialog = (
   element: HTMLElement,
   dialogParams: MediaPlayerBrowseDialogParams
 ): void => {
-  fireEvent(element, "show-dialog", {
-    dialogTag: "dialog-media-player-browse",
-    dialogImport: () =>
-      import(
-        /* webpackChunkName: "dialog-media-player-browse" */ "./dialog-media-player-browse"
-      ),
-    dialogParams,
-  });
+  fireEvent(element, "hass-media-browse", dialogParams);
```

Comment:
The media browser dialog should be lazy-loaded.

Reply:
I've updated the event listener to lazy load rather than defer load.

https://github.com/home-assistant/frontend/pull/7467/files#diff-5a8931fe0430bd4d3a2c6064914b55b4120c8de15c5f7a6280c9155cd60804a5R30